components/installer/pkg/kymahelm/kyma-helm-client.go

```diff
@@ -120,3 +128,14 @@ func (hc *Client) PrintRelease(release *release.Release) {
 	log.Printf("Status: %s", release.Info.Status.Code)
 	log.Printf("Description: %s", release.Info.Description)
 }
+
+// PrintOverrides .
+func (hc *Client) PrintOverrides(overrides string, releaseName string) {
+	log.Println("Overrides used for", releaseName)
```

Comment:
Please add information if it's an upgrade or install operation.
My proposal for the message:
"Overrides used for <releasename> [installation/update]"